less/navigation.less

```diff
@@ -1 +1,25 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+header {
+  border-bottom: 2px dashed @navigation-border;
+  padding: 30px 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-right: 1%;
+  margin-left: 19%;
```

Comment:
I'm assuming you were trying to center this content, since you are using `margin-right: 18%`, and `margin-left: 19%`, in that case, instead of writing both, I would write: `margin: 0 auto` and it would automatically center it. If you aren't trying to center it, than most likely, one of your margins is extra